lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -11,6 +11,10 @@ class Base
           ::RubyParser::SyntaxError,
         ]
 
+        DEFAULT_MASS_THRESHOLD = 28
+        BASE_POINTS = 1_500_000
+        POINTS_PER = 50_000
```

Comment:
@GordonDiggs I'm slightly worried that using inferred constants makes the code harder to follow for future devs. Now that you see it, any doubts?

If you're not worried, I'm not.